service/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/bean/ActivityMatchingIdsServiceTest.java

```diff
@@ -99,11 +99,6 @@ public void getMatchingIdsResponseNullMap() throws Exception {
         assertEquals(2, matchingIds.size());
     }
 
-    @Test
```

Comment:
Tried to fix this so that .init() was run as a part of every test, but then test would break other tests because we would replace a field that @InjectMocks had set with null. And since that field is private we can't set it, and I didn't feel like adding constructors or setters only for this test case.